### PR TITLE
[WIP] New layout

### LINKED ISSUE #50: Fix CSS for Computer/users on About Us Page
Seems to work fine for desktop but it's broken on widescreen and all of the mobile breakpoints.

### LINKED ISSUE #62: Font sizing
fonts are too big

Section Header:
40px

What we do -- subheading under icons:
25px

### LINKED ISSUE #64: "Lets Work Together" section
Change title font size:
60px

Drop us a Line button: 
Font size - 35px 
Button - width 350px, padding 10px

### LINKED ISSUE #65: About Section
Text needs to come in 200 px on either side
arrow keys should be resized approximately to scale (see img) and brought to level
text display should come up to bottom of monitor (see img) at ~50% opacity

image

### LINKED ISSUE #66: Link color needs to be set
image

Link color should be set to either #F06C7C or #60AED9

### LINKED ISSUE #68: Space BG Color is inconsistent
On index.html the spacebg has an overlay I didn't notice before. Instead of using an image tag with a transparent div above it, I am now using background-img with a linear gradient overlay.

index.html
image

blog.html
Before:
image

After:
image

### PR DESCRIPTION
**Pull request by Logan-Ruf**
Resolves #66, Resolves #64, Resolves #68, Resolves #62, Resolves #50, Resolves #65

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,6 +33,8 @@
 
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-128972978-1"></script>
+    <!--Press Start Font-->
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
```
Comment (ckcollab) [inline index.html]: I think we should import multiple google fonts in one call, can append it [here](https://github.com/ckcollab/ckcollab/blob/gh-pages/assets/css/global.css#L1) I think?

probably something like...
```
@import url("//fonts.googleapis.com/css?family=Montserrat:400|Press+Start+2P:400");
```

File: index.html [hunk 2]
```diff
@@ -85,7 +84,7 @@
 </div>
 
 <section id="services">
-    <div class="pink-stripe"></div>
+    <div class="pink-stripe"style="margin-top: 0 !important"></div>
```
Comment (ckcollab) [inline index.html:86]: we should avoid inline styling

File: index.html [hunk 3]
```diff
@@ -355,15 +353,18 @@
         position: relative;
         left: 80%;
         height: 380px;
-        top: -370px;
+        top: -320px;
     }
 
     .lets-work-together {
+        background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("../../assets/images/spacebg.jpg");
```
Comment (ckcollab) [inline index.html]:
```suggestion
        background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("../images/spacebg.jpg");
```

I think this path should work instead?

File: assets/css/global.css [hunk 4]
```diff
@@ -104,7 +106,7 @@
 /* Footer CSS */
 
 .footer {
-    background-image: url('../../assets/images/spacebg.jpg');
+    background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("../../assets/images/spacebg.jpg");
```
Comment (ckcollab) [inline assets/css/global.css:112]:
```suggestion
    background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("../images/spacebg.jpg");
```

another fix for this path, I think this makes more sense

File: assets/css/blog-landing.css [hunk 5]
```diff
@@ -53,7 +53,7 @@ section.heading {
 }
 
 .footer {
-    background-image: url('../images/spacebg.jpg');
+    background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("../../assets/images/spacebg.jpg");
```
Comment (ckcollab) [inline assets/css/blog-landing.css]:
```suggestion
    background-image: linear-gradient(rgba(21, 61, 165, .17), rgba(21, 61, 165, .17)), url("../images/spacebg.jpg");
```

**Review by ckcollab — COMMENTED** [review]
some quick comments

File: index.html (hunk 3 above)
Comment (sw00d) [reply index.html]: Your proposed path does not work, but perhaps this is what you were thinking `url("./assets/images/spacebg.jpg");`

**Pull request merged**